config/settings.yml

```diff
@@ -119,6 +119,14 @@ appeals_status:
   app_token: PUBLICDEMO123
   host: https://dsva-appeals-certification-dev-1895622301.us-gov-west-1.elb.amazonaws.com
 
+# Settings for (preneeds) burials.
+# TODO: enpoint needs to be set up for https and production api when available
+preneeds:
+  endpoint: http://qitcalpha03.cem.va.gov:8000/eoas_SOA/PreNeedApplicationPort
```

Comment:
i'm not sure we actually want to commit this URL in settings.yml -- anyone needing to use this should add these to their local versions of this file.